pkg/backends/azurekeyvault.go

```diff
@@ -29,59 +38,55 @@ func (a *AzureKeyVault) Login() error {
 
 // GetSecrets gets secrets from Azure Key Vault and returns the formatted data
 // For Azure Key Vault, `kvpath` is the unique name of your vault
+// For Azure use the version here not make really sens as each secret have a different version but let support it
```

Comment:
If you want an older version of a secret, there should be a way to specify that version but default to latest. So not sure this comment is needed.